homework/src/index.js

```diff
@@ -22,25 +23,114 @@
     Object.keys(options).forEach((key) => {
       const value = options[key];
       if (key === 'text') {
-        elem.innerText = value;
+        elem.innerHTML = value;
       } else {
         elem.setAttribute(key, value);
       }
     });
     return elem;
   }
 
+  const root = document.getElementById('root');
+  const div = createAndAppend('div', root, { class: 'data' });
+  const header = createAndAppend('p', div, { text: 'HYF Repositories', class: 'header' });
+  const select = createAndAppend('select', header, { id: 'list' });
+  const table = createAndAppend('table', root, { class: 'table' });
+  const tbody = createAndAppend('tbody', table);
+  const tr = createAndAppend('tr', tbody, { text: 'Repository: ', class: 'label' });
+
+  const repositoryLink = createAndAppend('a', tr);
+  const description = createAndAppend('tr', tbody, { class: 'label' });
+  const forks = createAndAppend('tr', tbody, { class: 'label' });
+  const container = createAndAppend('div', root, { id: 'container' });
+  const updated = createAndAppend('tr', tbody, { class: 'label' });
+
   function main(url) {
-    fetchJSON(url, (err, data) => {
-      const root = document.getElementById('root');
-      if (err) {
-        createAndAppend('div', root, { text: err.message, class: 'alert-error' });
-      } else {
-        createAndAppend('pre', root, { text: JSON.stringify(data, null, 2) });
-      }
-    });
-  }
+    fetchJSON(url)
+      .then(repositories => {
+        repositories.forEach((item, index) => {
+          createAndAppend('option', select, {
+            text: item.name,
+            value: index
+          });
+        });
 
+        repositories.sort((a, b) => a.name.localeCompare(b.name));
```

Comment:
UPDATE: I forgot to make this comment: there is a (sorting) mismatch between what you display in the select box and the repository you display. The entries in the select element are **not* sorted.

For instance, if I select the `JavaScript1` repo, I get the `CommandLine` repo. Did you see that too?